tests/Unit/Routes/ApiRoutesTest.php

```diff
@@ -204,6 +218,52 @@ public function testCollectOtherDuplicateVoucherRoute()
         ;
     }
 
+    public function testCollectUndeliveredNewVoucherRoute()
+    {
+        $created_at = Carbon::parse(config('arc.collect_delivery_date'))->addDay();
+        $this->vouchers[2]->created_at = $created_at;
+        $this->vouchers[2]->delivery_id = null;
+        $this->vouchers[2]->save();
+
+        $code = $this->vouchers[2]->code;
+
+        $payload = [
+            'transition' => 'collect',
+            'trader_id' => 1,
+            'vouchers' => [
+                $code,
+            ]
+        ];
+        $this->user->traders()->sync([1]);
+        $this->actingAs($this->user, 'api')
+            ->json('POST', route('api.voucher.transition'), $payload)
+            ->assertStatus(200)
+            ->assertJson(['warning' => trans('api.errors.voucher_unavailable')])
+        ;
+    }
+
+    public function testCollectUndeliveredOldVoucherRoute()
```

Comment:
Maybe re-name as something to do with collect undelivered voucher created before delivery in system? Old route vs new route confused me.

Reply:
will rename legacy

Reply:
Or just make the test title make sense